backend/src/yeomanui.ts

```diff
@@ -50,6 +51,7 @@ export class YeomanUI {
   private readonly replayUtils: ReplayUtils;
   private readonly customQuestionEventHandlers: Map<string, Map<string, Function>>;
   private errorThrown = false;
+  private startTime: number;
```

Comment:
i think that you do not need this param for usage analitics

Reply:
I need - Avital wants to know how much time takes to project to be generated.

Reply:
Done.